DSA/Hashmap/Hashmap.cpp

```diff
@@ -0,0 +1,113 @@
+#include<iostream>
+#include<cstdlib>
+#include<string>
+#include<cstdio>
+using namespace std;
+const int T_S = 200;
+class HashTableEntry {
+   public:
+      int k;
```

Comment:
It is Recommended to avoid use of single character variable name.

Reply:
resolved